src/EventListener/MailUsernameListener.php

```diff
@@ -62,7 +62,7 @@ public function saveMemberEmail($strValue, $dc)
             $this->connection->executeQuery('LOCK TABLES tl_member WRITE');
 
             // Check if the username already exists
-            $exists = $this->connection->fetchOne('SELECT TRUE FROM tl_member WHERE username = ?', [$strValue]);
+            $exists = $this->connection->fetchOne('SELECT TRUE FROM tl_member WHERE username = ? AND id != ?', [$strValue, $dc->id]);
```

Comment:
Doesn't it produce a warning in frontend with PHP 8? (see https://github.com/terminal42/contao-mailusername/issues/25)

Reply:
I suppose so - I'l check again today.